package.json

```diff
@@ -2,77 +2,77 @@
   "private": true,
   "scripts": {
     "start": "webpack-dev-server --config webpack.config.js",
-    "build": "webpack -p --config webpack_docker.config.js --display-error-details",
+    "build": "webpack -p --config webpack.config.js --display-error-details",
```

Comment:
Does this brake the container build?

Reply:
I noticed I was using two configurations with the exact same code for the start and the build script, so I deleted one and kept the other.